packages/admin/admin/src/common/Dialog.tsx

```diff
@@ -0,0 +1,112 @@
+import { Close } from "@comet/admin-icons";
+import {
+    ComponentsOverrides,
+    css,
+    // eslint-disable-next-line no-restricted-imports
+    Dialog as MuiDialog,
+    DialogProps as MuiDialogProps,
+    DialogTitle as MuiDialogTitle,
+    IconButton as MuiIconButton,
+    Theme,
+    useThemeProps,
+} from "@mui/material";
+import { ReactNode } from "react";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type DialogClassKey = "root" | "closeButton" | "dialogTitle";
+
+export type DialogProps = ThemedComponentBaseProps<{
+    closeButton: typeof MuiIconButton;
+    root: typeof MuiDialog;
+    dialogTitle: typeof MuiDialogTitle;
+}> & {
+    children?: ReactNode;
+    title?: ReactNode;
+    iconMapping?: {
+        closeIcon?: ReactNode;
+    };
+} & MuiDialogProps;
+
+type OwnerState = {
+    hasClose: boolean;
```

Comment:
I'd call this hasCloseButton

Reply:
Done in 7ac7a16b4dd68d292170c78ca69965f0ac6731e4